eth_pydantic_types/hash.py

```diff
@@ -87,23 +88,60 @@ def validate_size(cls, value: str) -> str:
         return validate_str_size(value, cls.size * 2)
 
 
-def _make_hash_cls(size: int, base_type: Type):
+class HashInt(int):
```

Comment:
Maybe we should just called these `Int` (w/o the Hash)

Reply:
We should probably remove Hash in a breaking change release from byte and str as well. It was only put there because I was copying `Hash32` from eth-typing module

https://github.com/ethereum/eth-typing/blob/main/eth_typing/evm.py#L12

but i agree it is confusing and doesnt really make sense